.github/workflows/docker.yaml

```diff
@@ -13,6 +13,22 @@ jobs:
     if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name == github.repository
     runs-on: ubuntu-20.04
     steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          # Check https://github.com/livepeer/go-livepeer/pull/1891
+          # for ref value discussion
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: Get build tags
+        id: build-tag
+        run: |
+          ./ci_env.sh echo '::set-output name=build-tags::$BUILD_TAGS'
+          set -x
+          echo "::set-output name=tags::$BUILD_TAGS"
```

Comment:
Actually, they are inconsistent, the first one [1] has `build-tags` while the second one has `tags`.

[1] I actually don't know what that first echo does. What is it for?